sdk/tests/conformance/resources/uniformBlockShader.vert

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Since this shader uses GLSL ES 3.0 syntax I wonder whether it needs the "#version 300 es" directive as the first line.